lib/interface/cli/helpers/helpers.unit.spec.js

```diff
@@ -32,6 +33,45 @@ describe('helpers unit tests', () => {
         }
     });
 
+    describe('general', () => {
+        describe('prepareKeyValueObjectsFromCLIEnvOption', () => {
+            it('should parse key value object from array', () => {
+                const result = prepareKeyValueObjectsFromCLIEnvOption([
+                    'SECRET_PASS=YWJjZA===',
+                    'USERNAME=testUserName',
+                ]);
+                expect(result).toEqual([
+                    {
+                        key: 'SECRET_PASS',
+                        value: 'YWJjZA===',
+                    },
+                    {
+                        key: 'USERNAME',
+                        value: 'testUserName',
+                    },
+                ]);
+            });
+
+            it('should parse key value object from string', () => {
+                const result = prepareKeyValueObjectsFromCLIEnvOption('TEST=testData');
+                expect(result).toEqual([
+                    {
+                        key: 'TEST',
+                        value: 'testData',
+                    },
+                ]);
+            });
+
+            it('should throw error when add incorrect environment variable', () => {
+                try {
+                    prepareKeyValueObjectsFromCLIEnvOption('TEST');
```

Comment:
We should use `expect.toThrow` instead, otherwise the test will pass it func will not throw. 
```suggestion
                    expect(() => prepareKeyValueObjectsFromCLIEnvOption('TEST')).toThrow(new CFError('Invalid environment variable format. please enter [key]=[value]'))
```